src/api/web.ts

```diff
@@ -44,3 +45,157 @@ export function getUserOrganizations(
 
     return res.status(200).json(orgs);
 }
+
+export function newUser(req: express.Request, res: express.Response): express.Response | void {
+  auth.newUser(req.body)
+    .then(auth => {
+      writeJsonFile(getAuthPath(), auth)
+        .then(() => {
+          return res.status(200).json({ message: 'User successfully added.' });
```

Comment:
don't want to interfere much, but `.json({ data: true / false })` would be more convenient to handle on the frond end. error or success messages does not belong into backend. my opinion.